app/jobs/create_gemini_recipient_ids_job.rb

```diff
@@ -0,0 +1,30 @@
+# Creates the Uphold Cards for a publisher
+class CreateGeminiRecipientIdsJob
+  include Sidekiq::Worker
+  sidekiq_options queue: :scheduler
+
+  def perform(gemini_connection_id:)
```

Comment:
```suggestion
  def perform(gemini_connection_id)
```
I don't think keyword arguments are supported by Sidekiq (I could be wrong)

Reply:
Done.